src/component/Multisearch/Multisearch.tsx

```diff
@@ -0,0 +1,344 @@
+import * as React from 'react';
+
+import NominatimSearch from
+  '@terrestris/react-geo/dist/Field/NominatimSearch/NominatimSearch';
+import { WfsSearchInput } from '@terrestris/react-geo';
+
+import './Multisearch.css';
+import { AutoComplete } from 'antd';
+import { LoadingOutlined } from '@ant-design/icons';
+
+import { transformExtent } from 'ol/proj';
+import OlFormatGeoJSON from 'ol/format/GeoJSON';
+
+import MapUtil from '@terrestris/ol-util/dist/MapUtil/MapUtil';
+
+// default props
+interface DefaultMultisearchProps {
+  className: string;
+  useNominatim: boolean;
+  useWfs: boolean;
+  minChars: number;
+  nominatimSearchTitle: string;
+  placeHolder: string;
+}
+
+interface MultisearchProps extends Partial<DefaultMultisearchProps> {
+  map: any;
+}
+
+interface MultisearchState {
+  searchTerm: string;
+  wfsResults: any[];
+  wfsFeatures: any[];
+  nominatimResults: any[];
+  nominatimFeatures: any[];
+  fetching: boolean;
+  options: any[];
+  searchAttributes: any;
+  searchConfig: any;
+  wfsSearchPending: boolean;
+  nominatimSearchPending: boolean;
+}
+
+/**
+ * Class representing the Multisearch.
+ *
+ * @class Multisearch
+ * @extends React.Component
+ */
+export default class Multisearch extends
+  React.Component<MultisearchProps, MultisearchState> {
+
+  public static defaultProps: DefaultMultisearchProps = {
+    className: 'multisearch',
+    useNominatim: true,
+    useWfs: true,
+    minChars: 3,
+    nominatimSearchTitle: 'Nominatim',
+    placeHolder: 'search location or data'
+  };
+
+  /**
+   * Create the Multisearch.
+   *
+   * @constructs Multisearch
+   */
+  constructor(props: MultisearchProps) {
+    super(props);
+
+    const searchLayers = MapUtil.getLayersByProperty(
+      this.props.map, 'searchable', true);
+    const searchConfig = {};
+    const searchAttributes = {};
+    searchLayers.forEach((l: any) => {
+      const conf = l.get('searchConfig');
+      searchConfig[conf.featureTypeName] = conf;
+      searchAttributes[conf.featureTypeName] = conf.attributes;
```

Comment:
can `conf` get `undefined` or `null` value (e.g. if layer is not configured as searchable)?